examples/bpmn-utlis.js

```diff
@@ -0,0 +1,577 @@
+// Inspired from 'Procurement Processes with Error Handling - Stencil Trisotech 3 pages.bpmn' of https://www.omg.org/cgi-bin/doc?dtc/10-06-02.zip
```

Comment:
:question: could we also extract all BPMN content from the remaining examples?
For now, we have the content that were duplicated, if we can do it everywhere this will simplify a lot.

Reply:
Done